appveyor.yml

```diff
@@ -0,0 +1,23 @@
+version: 2.0.{build}
+branches:
+  only:
+  - master
+image: Visual Studio 2017
+configuration: Release
+before_build:
+  appveyor-retry nuget restore
+build:
+  parallel: true
+  verbosity: minimal
+test:
+  assemblies:
+    - MetadataExtractor.Tests.dll
+test_script:
+  - dotnet test .\MetadataExtractor.Tests\MetadataExtractor.Tests.csproj --configuration Release --no-build
```

Comment:
As standard it was running `xunit.consol`-runner that was working for XmpCore because that the TFS for the test in XmpCore is `netcoreapp1.0;net451` and that runner only execute the classic net451 assembly tests.

In MetadataExtractor the test is only targeting `netcoreapp1.3` and not classic .net and to get the test to run I need to tell the appveyor test-runner to run the `dotnet test` instead.

Before I added the `test_script` property the build was failing, see example https://ci.appveyor.com/project/drewnoakes/metadata-extractor-dotnet/build/2.0.249#L397